KubeArmor/BPF/ka_ea_process_prog.bpf.c

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright 2021 Authors of KubeArmor
+
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+
+#include "maps.bpf.h"
+
+SEC("tracepoint/sys_enter_execve")
```

Comment:
in general, we used kprobe. is there any difference between tracepoint and kprobe?

Reply:
Tracepoints are static therefore more stable:

https://stackoverflow.com/a/47031794

https://blogs.oracle.com/linux/post/taming-tracepoints-in-the-linux-kernel

https://www.oreilly.com/library/view/linux-observability-with/9781492050193/ch04.html